plugins/orchestrator-common/src/permissions.ts

```diff
@@ -22,12 +22,7 @@ export const orchestratorWorkflowReadPermission = createPermission({
 });
 
 export const orchestratorWorkflowExecutePermission = createPermission({
-  name: 'orchestrator.workflow.execute',
-  attributes: {},
-});
-
-export const orchestratorWorkflowInstanceAbortPermission = createPermission({
-  name: 'orchestrator.workflowInstance.abort',
+  name: 'orchestrator.workflows',
```

Comment:
As mentioned by @batzionb, couldn't conditional rules work instead? I don't have much experience defining them. But they could work for what you are trying to achieve. 

From my basic understanding, you wound need to define a resource type, in this case something like `orchestrator-workflows` .Next, you would need to define your conditional rules, something like `hasWorkflowID`. And finally you would want to update your router to use the conditional rules.

I have not tried them out personally but I can supply some resources and a few plugins that currently use them at the moment. 

[Adding a resource permission check](https://backstage.io/docs/permissions/plugin-authors/03-adding-a-resource-permission-check), this includes information on support for conditional decisions

[Playlist plugin](https://github.com/backstage/community-plugins/blob/main/workspaces/playlist/plugins/playlist-backend/src/permissions/conditions.ts)

[Catalog plugin](https://github.com/backstage/backstage/tree/master/plugins/catalog-backend/src/permissions)

[Scaffolder plugin](https://github.com/backstage/backstage/blob/f899a829e09d88a2e37bcd93c9fdca919878a6b8/plugins/scaffolder-backend/src/service/rules.ts)